edenai_apis/apis/corticalio/info.json

```
@@ -0,0 +1,13 @@
+{
+  "text": {
+    "keyword_extraction": {
+      "constraints": {
+        "languages": [
+          "de",
+          "en"
+        ]
+      },
+      "version": "v1.1.0"
```

pls change to version 1.2.o
```suggestion
      "version": "v1.2.0"
```